tools/az_cli.py

```diff
@@ -100,6 +101,10 @@ def StartAnalysisVm(args: 'argparse.Namespace') -> None:
       logger.error('error: parameter --attach_disks: {0:s}'.format(
           args.attach_disks))
       return
+  if args.image_reference:
+    image_reference = json.loads(args.image_reference)
```

Comment:
This can throw a `JSONDecodeError` or a `TypeError`, and should be wrapped accordingly

Reply:
Done